crates/common/types/transaction.rs

```diff
@@ -39,7 +39,7 @@ pub struct LegacyTransaction {
     pub s: U256,
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Default)]
+#[derive(Clone, Debug, PartialEq, Eq, Default, Deserialize)]
```

Comment:
This struct has a custom `Serialize` implementation, directly deriving `Deserialize` here will make these traits incompatible and will lead to potential bugs down the line

Reply:
Fixed, custom Deserialize Trait was already implemented, forgot to remove the derive.